js/popupGallery.js

```diff
@@ -0,0 +1,74 @@
+import {picturesContainer} from './renderingPicture.js';
+
+const displayingWindow = document.querySelector('.big-picture');
+const closeBigPhoto = displayingWindow.querySelector('.big-picture__cancel');
+const bigPicture = displayingWindow.querySelector('.big-picture__img img');
+const likesCounter = displayingWindow.querySelector('.likes-count');
+const commentCount = displayingWindow.querySelector('.comments-count');
+const commentsList = displayingWindow.querySelectorAll('.social__comments li');
```

Comment:
Нам нужен контейнер в который мы будем помещать все наши комментарии, возьмем переменную `commentsList` для этого и просто туда достанем наш контейнер.
Так же мы возьмем один комментарий `.social__comment` и склонируем его (Это просто 1 комментарий из DOM но мы его склонировали и теперь будем использовать в качестве "шаблона" для создания каждого комментария)
```suggestion
const commentsList = displayingWindow.querySelector('.social__comments');
const commentTemplate = displayingWindow.querySelector('.social__comment').cloneNode(true);
```

Reply:
+